addons/mail/static/src/core/common/persona_model.js

```diff
@@ -77,8 +78,8 @@ export class Persona extends Record {
         if (this.type === "guest") {
             return imageUrl("mail.guest", this.id, "avatar_128", { unique: this.write_date });
         }
-        if (this.user?.id) {
-            return imageUrl("res.users", this.user.id, "avatar_128", { unique: this.write_date });
+        if (this.userId) {
+            return imageUrl("res.users", this.userId, "avatar_128", { unique: this.write_date });
```

Comment:
In diff but not because this PR, but this is not good. We should always use the avatar of the partner related to the user for having a standard url/cache for a single person.